### ISSUE TITLE
[Bug] Fix Zerion on Chrome/FF with new Zerion API

### ISSUE THREAD

**Issue by Tedko** [DimensionDev/Maskbook#3251]
## Bug Info

For now zerion data source is not working in Mask 

Zerion folks updated their API etc:

```
Vadim 👨‍🎤|Zerion, [11.05.21 18:44]
We released our JS library — https://github.com/zeriontech/defi-sdk-js.

Suji Yan, [11.05.21 18:45]
wow

Suji Yan, [11.05.21 18:45]
cool cc @differui

Vadim 👨‍🎤|Zerion, [11.05.21 18:45]
There is quite a lof of opend connection on your side. Can you check if you can close connections if your software do not perform requests?

Vadim 👨‍🎤|Zerion, [11.05.21 18:48]
We also usually close socket connection if a person leaves the tab — https://github.com/zeriontech/defi-sdk-js/blob/master/src/socket/page-visibility.ts

Suji Yan, [11.05.21 18:49]
added our dev
```

### How to reproduce?

1. change data source to zeiron
2. wallet detail info not loading

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [ ] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [ ] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

**Comment by Tedko** [comment]
since debank dont support Polygon assets will we need to have zerion ready. cc @guanbinrui

**Comment by Tedko** [comment]
this will also fix #2933

**Comment by Tedko** [comment]
almost work on chrome now. will wait for FF review
